### PR TITLE
restore dilib, which apparently is somehow still used in some spot-mapping jsp pages

### PR DESCRIPTION
**Pull request by naknomum**
this restores a library and `pom.xml` change, from [PR909](https://github.com/WildMeOrg/Wildbook/pull/909), which turned out to be still required by some wildbook functionality.

- revert partial change to `pom.xml`
- restore `local-repo/` dir, with the single library needed
- small note in a readme

### CODE REVIEW

**Review by TanyaStere42 — APPROVED** [review]
code looks good, restores everything and provides a solid explanation. waiting for deploy confirmation before merging

**Comment by naknomum** [comment]
running locally:
- tomcat is behaving as expected
- able to login and see a few pages
- `encounterSpotVisualizer.jsp` does not display jsp errors (as it did before with lib gone)

**Pull request merged**
